pygromos/files/gromos_system/gromos_system.py

```diff
@@ -157,13 +157,14 @@ def __init__(self, work_folder: str, system_name: str, in_smiles: str = None,
         self.checkpoint_path = None
         self.adapt_imd_automatically = adapt_imd_automatically
         self.verbose = verbose
-        
-        #GromosFunctionality
-        self._gromosPP_bin_dir = in_gromosPP_bin_dir
-        self._gromosXX_bin_dir = in_gromosXX_bin_dir
 
-        self._gromosPP = GromosPP(gromosPP_bin_dir=in_gromosPP_bin_dir)
-        self._gromosXX = GromosXX(gromosXX_bin_dir=in_gromosXX_bin_dir)
+        self._single_multibath = False
```

Comment:
Variables that formerly were static variables initialized with "None" were never initialized as member variables